frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -90,14 +92,19 @@ const MainLayoutContentContainer = styled('main')(({ theme }) => ({
 
 export const MainLayout = forwardRef<HTMLDivElement, IMainLayoutProps>(
     ({ children }, ref) => {
+        const newAdminUIEnabled = useUiFlag('adminNavUI');
         const { uiConfig } = useUiConfig();
+        const location = useLocation();
         const projectId = useOptionalPathParam('projectId');
         const { isChangeRequestConfiguredInAnyEnv } = useChangeRequestsEnabled(
             projectId || '',
         );
 
         const theme = useTheme();
         const isSmallScreen = useMediaQuery(theme.breakpoints.down('lg'));
+        const hideAdmin =
+            newAdminUIEnabled && location.pathname.indexOf('/admin') === 0;
```

Comment:
Needs some testing on sandbox, but we use a similar pattern elsewhere and since we're setting the basename in the root this should behave as expected here